fastkml/data.py

```diff
@@ -1,20 +1,32 @@
+from typing import Dict
 from typing import List
 from typing import Optional
 from typing import Tuple
 from typing import Union
 
 from typing_extensions import TypedDict
 
-import fastkml.config as config
 from fastkml.base import _BaseObject
 from fastkml.base import _XMLObject
+from fastkml.config import etree  # type: ignore[attr-defined]
 from fastkml.types import Element
 
 
 class SimpleField(TypedDict):
     name: str
     type: str
-    displayName: str  # noqa: N815
+    displayName: Optional[str]  # noqa: N815
+
+
+SimpleFields = List[Dict[str, Union[str, str]]]
```

Comment:
`Union[str, str]` does not make any sense to me

Reply:
Typo most likely, will fix.